### DISCUSSION TITLE
Welcome to statgen-courses Discussions!

### DISCUSSION THREAD

**Discussion by gaow** [statgenetics/statgen-courses#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lasophiea** [comment]
Hello, my name is Sophia and I'm a dietitian - nutritionist, with a master in Molecular Nutrition (aka Precision Nutrition). I recently completed my thesis on genetic predisposition in overweight/obesity. I would like to know more about how to get into statistical genetics and bioinformatics. Looking forward to talk to you and get some info. 
Thanks!

**Reply by gaow** [reply]
Hi Sophia, great to hear from you. This discussion board is meant for course related material -- if you want to discuss a particular subject related to our courses please post here the link to that course material and your questions. If you would like to communicate with one of our faculty please find our information [here](https://statgen.us/). Thanks!
